code/io/joy_ff-sdl.cpp

```diff
@@ -727,7 +727,7 @@ void joy_ff_afterburn_on()
 		SDL_HapticStopEffect(haptic, pAfterburn2.id);
 
 		pAfterburn2.eff.periodic.length = SDL_HAPTIC_INFINITY;
-		pAfterburn2.eff.periodic.magnitude = 0x3fff;
+		pAfterburn2.eff.periodic.magnitude = 0x1999;
```

Comment:
The effect is reused for the fly-by effect which uses a different magnitude so whenever the actual afterburner effect should be played the magnitude needs to be reset in case the fly-by effect was played before.